librocksdb-sys/build/system.rs

```diff
@@ -0,0 +1,57 @@
+fn verify_lib_dir<P: AsRef<std::path::Path>>(lib_dir: P) {
+    let lib_dir = lib_dir
+        .as_ref()
+        .canonicalize()
+        .unwrap_or_else(|_| panic!("Failed to canonicalize {:?}", lib_dir.as_ref()));
+    let dir_lst = std::fs::read_dir(&lib_dir)
+        .unwrap_or_else(|_| panic!("Failed to open library dir {:?}", lib_dir));
+    if dir_lst.count() == 0 {
+        panic!(
+            "Library dir {:?} is empty and is probably incorrect. Verify it is the correct path."
```

Comment:
You're not passing library name here, it won't be visible in panic message

Reply:
Good catch!